.github/workflows/ci.yml

```diff
@@ -13,10 +13,9 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 14
-          - 16
           - 18
           - 19
+          - 20
```

Comment:
Node.js 19 is also EOL and should be removed
Node.js 21 is current and should be added.

Those changes don't necessarily have to be in this PR however.